src/taipy/templates/multi-page-gui/hooks/post_gen_project.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import os
+
+import taipy
+
+with open(os.path.join(os.getcwd(), "requirements.txt"), "a") as requirement_file:
+    requirement_file.write(f"taipy == {taipy.version._get_version()}\n")
```

Comment:
Since the required Taipy version is generated on the fly, I'm using `==` here.
`>=` may not be necessary.